woocommerce/woocommerce-gutenberg-products-block
# Product Tag Cloud: Hide from Widgets screen in classic themes

Hide the Product Tag Cloud widget from the Widgets screen so users use the Tag Cloud block from Gutenberg instead.


Fixes #6088




### Screenshots
|Before|After|
|-|-|
|![bifor](https://user-images.githubusercontent.com/905781/165401663-a42302bf-4839-4626-9009-54685723c0b5.jpg)|![after-2](https://user-images.githubusercontent.com/905781/165401712-8484b92a-7805-46eb-8f9c-a8f301a1547a.jpg)|

### Manual Testing

How to test the changes in this Pull Request:
1. With a classic theme (ie: Storefront), go to Appearance > Widgets.
2. Open the block inserter and make sure the _Product Tag Cloud_ (isn't showing up)



### Changelog

> Hide the Product Tag Cloud from the Widgets screen in classic themes.
